README.md

```diff
@@ -1,50 +1,102 @@
 # merchandise-in-baggage
 
-**Who uses the repo/service**
+## Who uses the repo/service
 
-This is the backend service supporting two frontends (public facing and internally for administration through stride).
+merchandise-in-baggage backend service supports two frontends (public facing and internally for administration through
+stride).
 
-Both frontend services are maintained from the [merchandise-in-baggage-frontend](https://github.com/hmrc/merchandise-in-baggage-frontend) repository
-and are deployed as separate instances `merchandise-in-baggage-frontend` & `merchandise-in-baggage-admin-frontend`.
+Both `merchandise-in-baggage-frontend` & `merchandise-in-baggage-admin-frontend`
+services are maintained from
+the [merchandise-in-baggage-frontend](https://github.com/hmrc/merchandise-in-baggage-frontend) repository
+and are deployed as separate instances.
 
-**How to start the service locally**
+## Start service locally
 
-`sbt run` This will only start the service as standalone but unable to interact with any other services including DataBase
+`sbt run` This will only start the service as standalone but unable to interact with any other services
+
+To complete a journey locally run the services via [service manager 2](https://github.com/hmrc/sm2)
+with the following profile:
 
-To complete a journey locally run the services (minus the frontend) via [service manager 2](https://github.com/hmrc/sm2) with the following profile:
 ```bash
 sm2 --start MERCHANDISE_IN_BAGGAGE_ALL
-sm2 --stop MERCHANDISE_IN_BAGGAGE
 ```
 
-`local base url and port` http://localhost:8280/
+### Service Local URL
+
+For admin facing you will need to login through stride with the roles found in `application.conf`
 
-**How to run tests**
+```
+http://localhost:8281/declare-commercial-goods/start-import 
+``` 
+
+## How to run tests
 
 `./run_all_tests.sh` will run all the tests, including unit and contract tests. The contract tests will use
 contract files stored in the project root directory folder `pact` of both front-ends.
 The tests contract verifier can be executed by running the script:
 `checkincheck.sh`. Note - currently contracts test only runs for local build.
 
-
 The VerifyContractSpec test will pass locally if the pact test in merchandise-in-baggage-frontend is run first.
 The pact test in the frontend will populate the pact directory which it will use.
 
-**Endpoints**
-POST        /declarations       _to persist a declaration_                
+# Endpoints
```

Comment:
A few endpoints have been removed but please remove this `Endpoints` section if there is no valuable information. I don't see the point of this unless you want to add more info on the endpoints like in [here](https://github.com/hmrc/bc-passengers-declarations/blob/main/README.md#endpoints).